src/Invictus.Testing/LogicAppException.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Runtime.Serialization;
+using System.Security.Permissions;
+using GuardNet;
+
+namespace Invictus.Testing
+{
+    /// <summary>
+    /// Thrown when a problem occurs during interaction with a logic app running in Azure.
+    /// </summary>
+    [Serializable]
+    public class LogicAppException : Exception
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LogicAppException"/> class.
+        /// </summary>
+        public LogicAppException()
```

Comment:
I would remove this overload, we need to have logic app information as a minimum

Reply:
This is out of my hands. This is the correct way to implement an custom exception with properties + making it correctly serializable. Was not a feature-wise choice.

Reply:
Hm I can understand why you need this one but the others as well? I never do that because they allow you to create exceptions without essential information? 🤔

Reply:
It's ok for me if this is the way to go

Reply:
There's several things you have to do actually, serializable attribute, seriazible ctor, 3 ctors with message and inner exception, override getobjectdata to include custom props.

Reply:
Another reason to choose F# 😬